packages/documentation/src/stories/components/list/list.stories.ts

```diff
@@ -0,0 +1,45 @@
+import { StoryObj } from '@storybook/web-components';
+import { html, nothing } from 'lit';
+import { MetaComponent } from '@root/types';
+
+const meta: MetaComponent = {
+  id: '18566ca9-4502-4053-89f0-db5fdf6f906e',
+  title: 'Components/List',
+  tags: ['package:WebComponents'],
+  component: 'post-list',
+  parameters: {
+    badges: [],
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/JIT5AdGYqv6bDRpfBPV8XR/Foundations-%26-Components-Next-Level?node-id=558-7013&t=tiW8c2NHXI0mcJa5-1',
+    },
+  },
+  args: {
+    horizontal: null,
+    titleHidden: null,
+  },
+  argTypes: {
+    titleHidden: {
+      options: [true, false],
+    },
+    horizontal: {
+      options: [true, false],
+    },
```

Comment:
You do not need to manually add the properties to the argTypes, they are automatically created from your component.
```suggestion
```

However it would be good to add controls for the CSS custom properties!